src/app/shared/form/builder/ds-dynamic-form-ui/models/relation-group/dynamic-relation-group.component.html

```diff
@@ -1,14 +1,14 @@
 <a *ngIf="!(formCollapsed | async)"
    class="close position-relative"
-   ngbTooltip="{{'form.group-collapse-help' | translate}}"
+   ngbTooltip="{{'Collapse-help' | translate}}"
```

Comment:
This should be `{{'Click here to collapse' | translate}}`.

Reply:
thanks, fixed today